lessons/programming/python/functions/index.mdx

```diff
@@ -2,22 +2,24 @@
 title: Functions
 ---
 
+<CodeLabLayout title="PythonFunctions">
+
 # Functions
 Functions are known as building blocks of programs. A large problem can be divided into smaller functions, achieving modularity, reusability, and readability.
 
-- Essentially speaking, a function accepts some inputs, performs computations, and outputs the results.
+- A function accepts some inputs, performs computations, and outputs the results.
 - Functions can be called as many times as we want after they are defined.
 - Each function takes only one responsibility and is interdependent
 - Functions are logical, modular, and readable to human eyes. With functions as modules, it is easier to maintain the programs further.
 
 ## Built-in functions
-Built-in functions are pre-defined in and a part of Python languagne. For example, print(), range(), and type().
+Built-in functions are pre-defined in and a part of Python languagne such as print(), range(), and type().
```

Comment:
Typo:
pre-defined ~~in~~ and ~~a~~ part of the Python ~~languagne~~ -> **language**